scrapers.js

```diff
@@ -1692,14 +1692,13 @@ let scrapers = [
   {
     state: 'OH',
     country: 'USA',
-    url: 'https://odh.ohio.gov/wps/portal/gov/odh/know-our-programs/Novel-Coronavirus/welcome/',
+    url: 'https://coronavirus.ohio.gov/wps/portal/gov/covid-19/',
     scraper: async function() {
       let counties = [];
       let $ = await fetch.page(this.url);
       let $paragraph = $('p:contains("Number of counties with cases:")').text();
-      let regExp = /\(([^)]+)\)/;
-      let parsed = regExp.exec($paragraph);
-      let arrayOfCounties = parsed[1].split(',');
+      let parsed = $paragraph.replace(/([()])/g, '').replace('* Number of counties with cases: ','');
+      let arrayOfCounties = parsed.split(',');
```

Comment:
Is this backwards compatible with yesterday?

Reply:
yeah, tested against 3/15 scrapes. works fine.

Reply:
sigh, nevermind. On wrong branch.

Reply:
Thanks for dealing with it... I know it can be annoying to make the scrapers backwards compatible, but it's gonna pay off when we have timeseries data for all the things

Reply:
np, updated the PR. Now does the proper parsing. However 3/15 is broken, we don't seem to have a cache for Ohio? 3/14 and 3/16 work with the new code.

Reply:
Yes, we probably don't have 3/14. I will get it with the wayback machine once that work is compelte.